build.sh

```diff
@@ -52,6 +52,9 @@ dockerbuild()
     BUILD_COMMAND=/opt/code/build_projects/dotnet-cli-build/build.sh $DIR/scripts/dockerrun.sh --non-interactive "$@"
 }
 
+# Initialize the tooling
+$DIR/init-tools.sh "${args[@]}"
```

Comment:
We should think about how to manage this, perhaps the build.sh script inside dotnet-cli-build should make the call to init-tools.sh so it's already in docker.